### PR TITLE
add few mods, and deprecate some

### PR DESCRIPTION
**Pull request by art0007i**
the deprecated ones are part of the settings now.

also commit history of this pull is kinda weird but if you squash its fine I think

### CODE REVIEW

**Comment by art0007i** [comment]
ah wait developer category is gone. as of https://github.com/resonite-modding-group/resonite-mod-manifest/commit/f2e7d57bdf20aa96204f733961e95c9c8ec8defd
I wonder why this was done?

**Comment by XDelta** [comment]
Few mods used it previously, it was somewhat confusing as a category, and most if not all mods previously placed in it could be better placed in another category.

File: manifest/me.art0007i/MessageCopy/info.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "name": "MessageCopy",
+    "id": "me.art0007i.MessageCopy",
+    "description": "Allows you to copy all messages, not just text ones.",
+    "category": "Dashboard",
+    "sourceLocation": "https://github.com/art0007i/MessageCopy",
```
Comment (XDelta) [inline manifest/me.art0007i/MessageCopy/info.json:6]: 404? might be private

**Review by XDelta — CHANGES_REQUESTED** [review]

**Comment by art0007i** [comment]
well what category should I place it in then, it's pretty much only useful to developers.

File: manifest/me.art0007i/MessageCopy/info.json (hunk 1 above)
Comment (art0007i) [reply manifest/me.art0007i/MessageCopy/info.json:6]: should be good now. I forgot to create the repo at all actually oops

File: manifest/me.art0007i/JustBoundedUIX/info.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "category": "Technical Tweaks",
+    "sourceLocation": "https://github.com/art0007i/JustBoundedUIX",
+    "versions": {
+        "2.0.1": {
+            "releaseUrl": "https://github.com/art0007i/JustBoundedUIX/releases/tag/2.0.1",
+            "artifacts": [
+                {
+                    "url": "https://github.com/art0007i/JustBoundedUIX/releases/download/2.0.1/JustBoundedUIX.dll",
+                    "filename": "JustBoundedUIX.dll",
+                    "sha256": "96045eacd8afd70d5fa39c96f716412184f33e4d4e9de35ecc2205eda44596f0"
```
Comment (XDelta) [inline manifest/me.art0007i/JustBoundedUIX/info.json:14]: Hash doesn't appear to match

Comment (art0007i) [reply manifest/me.art0007i/JustBoundedUIX/info.json]: should be good now

**Review by XDelta — APPROVED** [review]

**Pull request merged**